django/db/migrations/recorder.py

```diff
@@ -18,23 +19,37 @@ class MigrationRecorder:
     If a migration is unapplied its row is removed from the table. Having
     a row in the table always means a migration is applied.
     """
-
-    class Migration(models.Model):
-        app = models.CharField(max_length=255)
-        name = models.CharField(max_length=255)
-        applied = models.DateTimeField(default=now)
-
-        class Meta:
-            apps = Apps()
-            app_label = "migrations"
-            db_table = "django_migrations"
-
-        def __str__(self):
-            return "Migration %s for %s" % (self.name, self.app)
+    _migration_class = None
 
     def __init__(self, connection):
         self.connection = connection
 
+    @classproperty
+    def Migration(cls):
+        """
+        When django.contrib.postgres in INSTALLED_APPS is in installed apps,
```

Comment:
The note should be generic -- describe the sort problem as it could happen to any app -- a reference to contrib.postgres isn't needed.